solr/solrj/src/java/org/apache/solr/client/solrj/impl/HttpSolrClientBuilderBase.java

```diff
@@ -167,10 +168,26 @@ public long getRequestTimeoutMillis() {
    */
   @SuppressWarnings("unchecked")
   public B useHttp1_1(boolean useHttp1_1) {
-    this.useHttp1_1 = useHttp1_1;
+    this.providedUseHttp1_1 = useHttp1_1;
     return (B) this;
   }
 
+  /**
+   * Return whether the HttpSolrClient built will prefer http1.1 over http2. This will be determined
```

Comment:
I think this is saying too much honestly.  Docs need to be maintained.  It exposes little details and possibly repeats things elsewhere.  *every* getter (lets say this is a getter), will use what is provided if the user supplies a choice.  Let's not say the obvious, since then we'll be tempted to repeat this again and again and again...   
Secondly, any getter like method (includes this) on a builder is really for internal use, so I wouldn't bother documenting it.  Thirdly, if you want to document the semantics of what this settings means, the place to do that is on the setter (or "with" method).  It's already documented.  So just drop this IMO.